### PR TITLE
docs: Add examples

### PR DESCRIPTION
**Pull request by maelle**
@krlmlr is this the sort of examples you're after when you mention dummy methods? Once this is clarified, I can add more.

### CODE REVIEW

File: R/relational-rel.R [hunk 1]
```diff
@@ -162,9 +162,10 @@ rel_limit <- function(rel, n, ...) {
 #' @inheritParams rel_to_df
 #' @return a new relation object with distinct rows
 #' @export
-#' @examplesIf FALSE
-#' rel <- rel_from_df(mtcars)
-#' rel2 <- rel_distinct(rel)
+#' @examples
+#' mtcars_rel <- new_relational(mtcars)
```
Comment (krlmlr) [inline R/relational-rel.R]:
```suggestion
#' mtcars_rel <- new_relational(mtcars, class = "xxx")
```

File: R/relational-rel.R [hunk 2]
```diff
@@ -162,9 +162,10 @@ rel_limit <- function(rel, n, ...) {
...
+#' rel_distinct.relational <- function(df) unique(df)
```
Comment (krlmlr) [inline R/relational-rel.R]:
```suggestion
#' rel_distinct.xxx <- function(df) unique(df)
```

**Review by krlmlr — COMMENTED** [review]
Thanks, yes, something like this.

**Comment by krlmlr** [comment]
Did you mean to add `class = "..."` arguments?

**Comment by maelle** [comment]
Is it a problem that the dummy method don't preserve the class? Probably, about to fix that now :sweat_smile:

**Comment by maelle** [comment]
The examples feel very contrived. Still missing are

-  rel_order
- rel_join
- rel_filter
- rel_project
- rel_aggregate

For them, the examples would probably be even dumber. Is that ok? Or should we have a single Rd page for them all, with an example showing only pseudocode (maybe in a dontrun)?

**Comment by krlmlr** [comment]
Thanks! I'll take it from here.

**Pull request merged**